resources/examples/multi_icebergs_cluster/inventory/group_vars/all/j2_variables/internal_variables.yml

```diff
@@ -1,9 +0,0 @@
-### NAMING
-
-iceberg_naming: iceberg
-equipment_naming: equipment
-management_networks_naming: ice
-master_groups_naming: mg
-
-managements_group_name: mg_managements
-
```

Comment:
I believe we should keep this file in the user's inventory to keep the configuration in a single place.